tests/test_utils.py

```diff
@@ -0,0 +1,14 @@
+from models import PullRequest
+import json
+
+class MockPullRequest(PullRequest):
+    def __init__(self, labels_json_file):
+        PullRequest.__init__(self)
```

Comment:
you should call `super(MockPullRequest, self).__init__()` here instead

Reply:
Got it. What's the difference? I had tried something close to that which wasn't work but I forget exactly what

Reply:
https://docs.python.org/2/library/functions.html#super
What you did is essentially the same in this case, but by convention we use `super` so that the code is more maintainable and can easily be used with Python's multiple inheritance.